src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -192,9 +192,6 @@ h1, h2, h3, h4 {
 .rustdoc-breadcrumbs {
 	grid-area: main-heading-breadcrumbs;
 	line-height: 1.25;
-	display: flex;
```

Comment:
As far as I can see, this is the only useful change.

Reply:
z-index from a containing block on the parent + overflowing padding of inline `a` did the trick.